package.json

```
@@ -13,8 +14,8 @@
   },
   "license": "MIT",
   "files": [
-    "index.js",
-    "lib"
+    "index.ts",
+    "build"
   ],
   "main": "index.js",
```

```suggestion
  "main": "build/index.js",
```
This is what node should grab when you `require('strip-comments')`